src/handlers/logging.js

```diff
@@ -1,5 +1,10 @@
-import { configureLoggingService } from '@edx/frontend-logging';
+import { configureLoggingService, NewRelicLoggingService } from '@edx/frontend-logging';
 
+/* eslint-disable no-param-reassign */
 export default async function logging(app) {
```

Comment:
Could you use defaults here?

`function logging({ loggingService = NewRelicLoggingService }) {`

Reply:
Hmm.  I'd still have to assign it back to app to ensure that other handlers that need it get the right thing.  I think in this case the `if` might be more readable, just because it's all clearly hanging off the `app` variable. ¯\_(ツ)_/¯  But yeah, I kind of could. Worth a thought. :)

Reply:
Got it. This readable anyway, just a passing thought.